index.html

```diff
@@ -9,37 +9,41 @@
 </head>
 <body>
     <div class="container">
-
         <div class="todo">
-            <header>
+            <header class="todo__header">
                 <h1>Todos</h1>
             </header>
 
-            <input type="text" name="new-todo" id="new-todo" placeholder="What needs to be done?">
-            <label class="small">
-                    <input type="checkbox" name="all-complete-todo" id="all-complete-todo">
-                    Mark all as complete
-            </label>
+            <div class="todo__content">
+                <input type="text" name="todo__new-todo-field" class="todo__new-todo-field" placeholder="What needs to be done?">
```

Comment:
the `name` field wont follow BEM naming conventions. You'll also run into cases where you do the javascript class selectors  such as `js-new-todo-input` and these wont follow BEM either. The only thing that does is actual CSS styling.